tests/JustSaying.UnitTests/Messaging/Channels/Fakes/FakeAmazonSqs.cs

```diff
@@ -48,6 +49,11 @@ public Task<string> AuthorizeS3ToSendMessageAsync(string queueUrl, string bucket
         return Task.FromResult(new AddPermissionResponse());
     }
 
+    public Task<CancelMessageMoveTaskResponse> CancelMessageMoveTaskAsync(CancelMessageMoveTaskRequest request, CancellationToken cancellationToken = new CancellationToken())
```

Comment:
We should change all these optional parameters to use the simplified syntax:

```suggestion
    public Task<CancelMessageMoveTaskResponse> CancelMessageMoveTaskAsync(CancelMessageMoveTaskRequest request, CancellationToken cancellationToken = default)
```